### DISCUSSION TITLE
Simple API for lights and switches

### DISCUSSION THREAD

**Discussion by maximehk** [GrandMoff100/HomeAssistantAPI#126] [Ideas]
Hi,

I'm pretty new to your library but I wanted to share something I've used it for.

I've added an example and source code here: https://github.com/maximehk/ha_lights

The idea is to make a very simple high level interface to control lights and switches :)

Thanks a lot for providing this awesome package 😀

Hope you like it !

Max

**Comment by GrandMoff100** [comment]
I'm glad you love my package!!

From what I've seen of your source code examples it looks pretty cool!

Maybe one day it could get faster one day with a library that wraps the web socket api instead. It would probably be able to control the lights faster with a persistent web socket connection that individual request specific connections. But that's another discussion.
